source/includes/api/_subscriptions.md

```diff
@@ -500,14 +501,14 @@ curl -X PATCH https://api.resourcewatch.org/v1/subscriptions/:id/send_confirmati
 
 ## Modify subscription
 
-To modify a subscription:
-
 ```shell
 curl -X PATCH https://api.resourcewatch.org/v1/subscriptions/:id \
 -H "Authorization: Bearer <your-token>"
 ```
 
-With a body with the same format as before.
+To modify a subscription, use the following PATCH endpoint. The body should be formatted in the same way as when creating a subscription.
```

Comment:
avoid piggibacking off other docs - they may change, and that will propagate here in a way that's hard to detect and fix until someone flags this as a bug.

Reply:
also: add permissions info. Who can update a subscription?

Reply:
Fixed the piggybacking and added permissions info :+1: